tests/cpp_unit_tests/test_main_model.cpp

```diff
@@ -960,6 +960,110 @@ TEST_CASE_TEMPLATE("Test main model - restore components", settings, regular_upd
     }
 }
 
+TEST_CASE_TEMPLATE("Test main model - updates w/ alternating compute mode", settings, regular_update, cached_update) {
+    State state;
+    auto main_model = default_model(state);
+
+    auto const math_output_orig = main_model.calculate_power_flow<true>(1e-8, 20, CalculationMethod::linear);
+
+    ConstDataset const update_data{
+        {"sym_load", ConstDataPointer{state.sym_load_update.data(), static_cast<Idx>(state.sym_load_update.size())}},
+        {"asym_load",
+         ConstDataPointer{state.asym_load_update.data(), static_cast<Idx>(state.asym_load_update.size())}}};
```

Comment:
please instead use `shunt` instead of `(a)sym_load` because a load/gen does not affect ybus and therefore always returns `UpdateChange{false, false}` as you can see in https://github.com/PowerGridModel/power-grid-model/blob/main/power_grid_model_c/power_grid_model/include/power_grid_model/component/load_gen.hpp#L95 , as opposed to `shunt` that does cause a parameter update (but no topology update!!!) as you can see in https://github.com/PowerGridModel/power-grid-model/blob/main/power_grid_model_c/power_grid_model/include/power_grid_model/component/shunt.hpp#L56 , if and only if one or more of the new values are not identical to the original ones